glade_forms/analysis_options.yaml

```diff
@@ -5,3 +5,11 @@ dart_code_metrics:
     - package:netglade_analysis/dcm.yaml
   pubspec-rules:
     prefer-publish-to-none: false
+  rules:
+    prefer-single-declaration-per-file: false
+    avoid-commented-out-code: false # code examples in comments
+    prefer-boolean-prefixes: false # TODO
+    prefer-abstract-final-static-class: false # TODO
+    prefer-named-parameters: false # TODO
+    dispose-class-fields: false # TODO
+    avoid-undisposed-instances: false # TODO
```

Comment:
- avoid-undisposed-instances: ignored-invocations settings https://dcm.dev/docs/rules/flutter/avoid-undisposed-instances
- dispose-class-fields: ignore comment, i would type there something like that "callback parameters are better without named"
- prefer-abstract-final-static-class: fix
- prefer-boolean-prefixes: ok to disable (breaks API), or maybe just ignore comments on line/file?
- avoid-commented-out-code: above the comment add "// TODO(alternative): you can also do this" or something like that, to disable the warning
- prefer-single-declaration-per-file: add ignore for the one case